### PR TITLE
add nvompi toolchain definition (NVHPC + OpenMPI)

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — APPROVED** [review]
lgtm

File: easybuild/toolchains/nvompi.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for nvompi compiler toolchain (NVHPC + Open MPI).
+
+:author: Maxime Boissonneault (Universite Laval, Calcul Quebec, Compute Canada)
+:author: Bart Oldeman (McGill University, Calcul Quebec, Compute Canada)
```
Comment (boegel) [inline easybuild/toolchains/nvompi.py:29]: @robert-mijakovic Please change author to yourself?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by ocaisa** [comment]
@robert-mijakovic This looks good to merge, just need to address the comment

**Comment by robert-mijakovic** [comment]
> @robert-mijakovic This looks good to merge, just need to address the comment

Done. Thanks for reminding me.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**